source/blog/2025-10-08-update-on-the-orm-2-end-of-life.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "An update on the ORM 2 End of Life"
+authorName: Benjamin Eberlei
+authorEmail: kontakt@beberlei.de
+permalink: /2025/10/08/an-update-on-the-orm-2-end-of-life.html
+---
+
+As part of our Core team meetup in Paris this October 2025, we discussed what the next steps are for ORM 2 and its end of life. 
+
+ORM 3 was released almost 2 years ago, and we initially planned with the ORM 2 EOL for February 2026. 
+
+So far adoption of ORM 3 is about 25-30% of ORM 2 based on Packagist installation numbers, which is not a big enough number. DBAL adoption of major 3 is much higher at roughly 60%.
+
+Therefore we are going to push out the ORM 2 End of Life to at least February 2027, but we will restrict what kind of changes are eligible for the 2.0 branch further.
```

Comment:
This should talk about `2.x` branch, not `2.0` (no changes are allowed in the `2.0` branch. It does not even exist anymore).